internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -200,6 +205,15 @@ func (opts *Opts) Run() error {
 	return nil
 }
 
+func (opts *Opts) clusterError(err error) error {
```

Comment:
`opts` seems unused
```suggestion
func clusterError(err error) error {
```

Reply:
also is this function really needed? it only wraps an `if`

Reply:
had to do it because of linter `cyclomatic complexity 16 of func `(*Opts).Run` is high (> 15) (gocyclo)`

Reply:
instead of having clusterError I'll wrap dbuser creation, accesslist creation and cluster creation into another method